### PR TITLE
add swap infos in the validate on device screen

### PR DESCRIPTION
**Pull request by machard**
https://ledgerhq.atlassian.net/browse/LL-3572

### CODE REVIEW

File: src/renderer/components/DeviceAction/rendering.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
 // @flow
 import React, { useCallback } from "react";
+import _ from "lodash";
```
Comment (gre) [inline src/renderer/components/DeviceAction/rendering.js:3]: Usually we don't import _ but we will import the fn style (e.g. leash/map)

File: src/renderer/modals/Swap/SwapBody.js [hunk 2]
```diff
@@ -19,14 +20,16 @@ type SwapSteps = "summary" | "device" | "finished";
...
+  status,
   onClose,
   onStepChange,
   onCompleteSwap,
   activeStep,
   ratesExpiration,
 }: {
   swap: { exchange: Exchange, exchangeRate: ExchangeRate },
-  transaction: any, // FIXME
+  transaction: Transaction, // FIXME
```
Comment (gre) [inline src/renderer/modals/Swap/SwapBody.js:31]: I guess this makes it no longer a FIXME

**Review by gre — CHANGES_REQUESTED** [review]

**Review by gre — DISMISSED** [review]

**Review by valpinkman — DISMISSED** [review]

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
Typo is incorrect (too big):

Actual:
image

Expected:
image

**Comment by cthepot-ledger** [comment]
@machard the font isnot the same on the second screen. Kindly amend for consistency 
Screenshot 2020-12-23 at 13 04 48

Screenshot 2020-12-23 at 13 04 32

**Review by Arnaud97234 — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]

**Pull request merged**